src/graphics.rs

```diff
@@ -1320,6 +1322,7 @@ fn quad_indices() -> [u16; 6] {
     [0, 1, 2, 0, 2, 3]
 }
 
+/// A wrapper over the Index type used specifically for Sprites
```

Comment:
the `Index` type is not exposed, this type *is* the index of a `Sprite` in a `SpriteBatch`